lib/client.js

```diff
@@ -3,7 +3,7 @@
 require('object.assign').shim();
 var request = require('request');
 
-var defaultBaseUrl = 'http://api.hubapi.com/';
+var defaultBaseUrl = 'https://api.hubapi.com/';
```

Comment:
Could you just remove the dash here and keep the rest the same instead?

Reply:
I agree, I think this is the better option for clarity

Reply:
Agreed, changes in incidents should be kept to a minimum because we're acting under pressure.

Reply:
updated